In [4]:
name = "Alex"
hourly_wage = 22.50
hours = 40

weekly_pay = hourly_wage * hours

print(f"{name} worked {hours} hours at ${hourly_wage:.2f} an hour for a weekly pay of ${weekly_pay:.2f}.")

Alex worked 40 hours at $22.50 an hour for a weekly pay of $900.00.


In [5]:
temps = [61, 65, 72, 58, 70, 74, 68]
warm_count = 0

for temp in temps:
    if temp > 65:
        warm_count += 1

print(f"Number of warm days above 65: {warm_count}")

Number of warm days above 65: 4


In [6]:
def to_celsius(f):
    return (f - 32) * 5 / 9

celsius_value = round(to_celsius(temps[0]), 1)
print(celsius_value)


16.1


In [7]:
movie = {
    "title": "Inception",
    "year": 2010,
    "rating": 8.8
}

for key, value in movie.items():
    print(f"{key}: {value}")

title: Inception
year: 2010
rating: 8.8


In [9]:
import pandas as pd

df = pd.read_csv('chickwts.csv', index_col=0)

display(df.head())
print(df.shape)
display(df['weight'].describe())

,weight,feed
1,179,horsebean
2,160,horsebean
3,136,horsebean
4,227,horsebean
5,217,horsebean


(71, 2)


count     71.000000
mean     261.309859
std       78.073700
min      108.000000
25%      204.500000
50%      258.000000
75%      323.500000
max      423.000000
Name: weight, dtype: float64

Output from the describe() function reveals that the mean chick weight is around 261.3 grams and the standard deviation is roughly 78.1 grams. The minimum chick weight is 108 grams, while the maximum chick weight is 423 grams.